packages/webview_flutter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: webview_flutter
 description: A Flutter plugin that provides a WebView widget on Android and iOS.
-version: 0.3.19+5
+version: 0.3.19+6
```

Comment:
I wonder whether we should consider this a breaking change (as it will break any app relying on these links not being opened)